jest.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     '!src/components/Icons/Arrows/**/*.{ts,tsx}',
     '!src/components/Icons/Information/**/*.{ts,tsx}',
     '!src/components/Icons/Essencials/**/*.{ts,tsx}',
+    '!src/components/Icons/Document/**/*.{ts,tsx}',
```

Comment:
🔧 Os outros diretórios estão no plural, acho que aqui deveria seguir o mesmo padrão
```suggestion
    '!src/components/Icons/Documents/**/*.{ts,tsx}',
```

Reply:
I already talked to morgs and made the changes here https://github.com/magnetis/astro-native/pull/577/commits/c27f9b2e8d5b2c4c90e72a71f1067f0799938707